homework_10/task2.py

```diff
@@ -0,0 +1,7 @@
+def ismax(*numbers):
+    return max(numbers)
```

Comment:
ზოგადად, ფუნქციას იმისთვის ვქმნით რომ შიგნით ლოგიკა დავწეროთ და ფუნქციაში შიგნით მხოლოდ მზა ფუნქცია თუ გამოვიყენეთ 1 ხაზისთვის ამ ფუნქციის შექმნა არამართებულია, გთავაზობ ალტერნატივას:
def my_max(*args):
_max = args[0]
for i in args:
if type(i) == int or type(i) == float:
if i >_max:
_max = i
return _max

print(my_max(1,2,"car",-7.5,25,15,27))